src/ContentClient.ts

```diff
@@ -106,9 +105,7 @@ export class ContentClient implements ContentAPI {
     // Check if we are running in node or browser
     const areWeRunningInNode = isNode()
 
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    const form: FormData = areWeRunningInNode ? new NodeFormData() : new FormData()
+    const form = new FormData()
```

Comment:
how is it that this works now?

Reply:
FormData works both for browsers and node.js https://github.com/form-data/form-data/blob/master/lib/browser.js